src/style/colors.js

```diff
@@ -81,22 +77,52 @@ class Colors {
         return color;
       }
       return requiredColor;
-    } else { // Handles dynamic colors (non uilib colors)
-      let tintLevel = Math.floor(Number(tintKey) / 10);
-      tintLevel = Math.max(1, tintLevel);
-      tintLevel = Math.min(8, tintLevel);
-      
-      if (tintLevel === BASE_COLOR_LEVEL) {
-        return color;
-      } else if (tintLevel <= BASE_COLOR_LEVEL) {
-        const darkRatio = darkRatios[tintLevel - 1];
-        return Color(color).darken(darkRatio).hex();
-      } else {
-        const lightRatio = lightRatios[tintLevel - 4];
-        return Color(color).mix(Color('#ffffff'), lightRatio).hex();
-      }
+    } else { 
+      // Handles dynamic colors (non uilib colors)
+      let tintLevel = Math.floor((Number(tintKey) / 10) + 1);
```

Comment:
why the `+ 1` here?

Reply:
to match the 10 - 80 values to my 20 - 90 range